examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -120,14 +120,15 @@ export async function POST(req: Request) {
               isFunctionCall = true
             }
           }
-          if (!isFunctionCall && output.name == 'final-output') {
+          if (!isFunctionCall && output.name == 'answer') {
```

Comment:
this workflow was redeployed without notice, we should probably pin to a release tag